
For some reason tar generates some broken tar when called with the expanded
pathes ...

  - Rene Rebe <rene@exactcode.de>

--- ./Makefile.orig	2005-02-26 22:42:28.913980080 +0100
+++ ./Makefile	2005-02-26 22:42:43.256799640 +0100
@@ -199,7 +199,7 @@
 	# Remove getopt header since shadow password support is disabled.
 	$(RM) include/shadow.h
 endif
-	tar -chf - include/* | tar -xf - -C $(PREFIX)$(DEVEL_PREFIX);
+	tar -chf - include/ | tar -xf - -C $(PREFIX)$(DEVEL_PREFIX);
 	-@for i in `find  $(PREFIX)$(DEVEL_PREFIX) -type d` ; do \
 	    chmod 755 $$i; chmod 644 $$i/*.h > /dev/null 2>&1; \
 	done;
